experiments/templatetags/experiments.py

```diff
@@ -190,8 +190,12 @@ def experiment_enrolled_alternative(context, experiment_name):
 def _experiment_enrolled_alternative(context, experiment_name):
     request = context.get('request', None)
     user = participant(request=request)
-    return user.get_alternative(experiment_name, request)
-
+    alternative = user.get_alternative(experiment_name, request)
+    if user.active_experiment:
+        request.experiments_exposure = {
+            'experiment_name': user.active_experiment,
```

Comment:
How are we handling the case when there are multiple active experiments? I'd make this a list of dicts, or a dict with experiment names as keys.